go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/influxdata/cron v0.0.0-20201006132531-4bb0a200dcbe
 	github.com/influxdata/flux v0.125.0
 	github.com/influxdata/httprouter v1.3.1-0.20191122104820-ee83e2772f69
-	github.com/influxdata/influx-cli/v2 v2.0.0-20210713195937-a69f06b41b45
+	github.com/influxdata/influx-cli/v2 v2.1.1-0.20210813175002-13799e7662c0
```

Comment:
Is updating this required for the tests to pass or something like that, with the API change?

Reply:
The existing backup/restore e2e test passes both ways, since we left the old route in as deprecated. Bumping this makes sure we're testing the "official" API.